packages/fixie-sdk/src/request-wrapper.tsx

```diff
@@ -75,33 +76,56 @@ export function FixieRequestWrapper({ children }: { children: AI.Node }, { getCo
   }
   const { request } = requestContext;
 
-  // If we're using OpenAI (or default), enable the OpenAI proxy.
-  if (!request.generationParams?.modelProvider || request.generationParams.modelProvider.toLowerCase() === 'openai') {
-    wrappedNode = (
-      <OpenAI
-        client={
-          new OpenAIClient({
-            apiKey: authToken,
-            baseURL: new URL('api/openai-proxy/v1', apiBaseUrl).toString(),
-            fetch: globalThis.fetch,
-          })
-        }
-        chatModel={(request.generationParams?.model as ValidChatModel | undefined) ?? 'gpt-3.5-turbo'}
-      >
-        {wrappedNode}
-      </OpenAI>
-    );
-  } else {
-    throw new Error(`The model provider ("${request.generationParams.modelProvider}") is not supported.`);
+  const modelProvider = request.generationParams?.modelProvider?.toLowerCase() ?? 'openai';
+  if (modelProvider !== 'openai' && modelProvider !== 'anthropic') {
+    throw new Error(`The model provider ("${request.generationParams?.modelProvider}") is not supported.`);
   }
 
-  return (
+  // Set both OpenAI and Anthropic Clients, but only configure the requested one as the default ChatCompletion model.
```

Comment:
So now if `children` doesn't specify which provider to use, it knows which one to take because one of them doesn't have `chatModel` set?
I'm just worried that the default would become the inner-most one (i.e. Anthropic).

Reply:
Basically the props are additive:
- `client` -- sets the anthropic/openai client
- `chatModel` -- sets the default chat model
- `completionModel` -- sets the default completion model

The AI.JSX change is that we can now set `client` without also setting `chatModel`